src/styles/block/card.scss

```
@@ -0,0 +1,68 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 16px;
+  width: 200px;
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
```

just use margin-bottom. Check cases below
```suggestion
    margin-bottom: 4px;
```